web/cobrands/sass/_base.scss

```diff
@@ -182,6 +182,11 @@ a {
   }
 }
 
+.meta-2 a[name] {
+    padding-top: 5em;
+    margin-top: -5em;
```

Comment:
I think you’ll also need `display: block` if you want this vertical margin rule to have any effect. (Vertical margin is ignored on inline elements.)